.github/workflows/large_scale_performance_tests.yml

```diff
@@ -0,0 +1,137 @@
+name: Large Scale Performance Tests
```

Comment:
Why do we need a different workflow to run this? Can we reuse the `locust_performance_test` workflow? 
The extra step can be done using lifecycle hook in the locust file, if it does not take too long to run